packages/checkbox/src/Checkbox.story.tsx

```diff
@@ -32,18 +32,21 @@ storiesOf('Checkbox', module)
     const darkMode = boolean('darkMode', false);
 
     return (
-      <Checkbox
-        darkMode={darkMode}
-        disabled={boolean('Disabled', false)}
-        indeterminate={boolean('Indeterminate', false)}
-        label={text('Label', 'I agree to this thing.')}
-        bold={boolean('Bold', false)}
-        animate={boolean('Animate', true)}
-        className={css`
-          padding: 20px;
-          background-color: ${darkMode ? uiColors.gray.dark3 : uiColors.white};
-        `}
-      />
+      <LeafygreenProvider>
```

Comment:
Curious why you removed bold?

Reply:
Doesn't look like the Figma component supports a `bold` prop anymore

Reply:
Seems like there is a wider desire to `unbold` the Label component, but that should be done in a wider ticket
https://jira.mongodb.org/browse/PD-1689